components/UI/RangeHistogram/RangeHistogram.module.css

```diff
@@ -1,5 +1,10 @@
 .histogram {
-    padding: 16px 15px 0;
+    --histogram-padding: 10px;
```

Comment:
maybe move this variable to globals.css? some other historgram variables are defined in colors.css but this variable is not about color

Reply:
Moved another global variables to histogram styles)

Global values must have shared tokens. Histogram this is a local component.